app/build.gradle

```diff
@@ -47,13 +47,21 @@ dependencies {
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'com.vatsal.imagezoomer:image-zoomer:1.0.2'
+
     implementation 'com.borjabravo:readmoretextview:2.1.0'
+
     implementation 'com.borjabravo:readmoretextview:2.1.0'
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'com.squareup.retrofit2:retrofit:2.6.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.6.1'
     implementation 'com.google.firebase:firebase-storage:9.2.1'
     implementation 'com.google.firebase:firebase-auth:9.2.1'
 
+    implementation 'com.squareup.retrofit2:converter-gson:2.6.1'
+    implementation 'com.squareup.retrofit2:retrofit:2.6.1'
+    implementation 'com.github.bumptech.glide:glide:4.9.0'

```

Comment:
Also here - pay attention to the previous lines, you already have what's necessary.